test/compare-acorn-es6/class-declaration.js

```diff
@@ -0,0 +1,9 @@
+class ComputedKey {
+    [n1 + n2]() {}
+}
+class ComputedKeyWithParenthesis {
+    [n1 = (n2 + n3) * n4]() {}
```

Comment:
```suggestion
class AssignmentExpressionAsKey {
    [n1 = n2]() {}
```

Reply:
https://github.com/estools/escodegen/pull/415/commits/25eb2b8f14ad4bf7c06030751f5b8f1d13dce0eb 
Thank you